app/ui/lib/Badge.tsx

```diff
@@ -62,3 +62,13 @@ export const Badge = ({
     </span>
   )
 }
+
+export const AffinityGroupPolicyBadge = (props: { policy: string; className?: string }) => (
+  <Badge
+    color="neutral"
+    variant={props.policy === 'allow' ? 'default' : 'solid'}
```

Comment:
use `ts-pattern` for an exhaustive match on the policy values. also maybe move this into the file where it's used, or if it's going to be shared, maybe `app/components`